### DISCUSSION TITLE
GitHub team could not be created. There seems to be a problem on github.com, please try again.

### DISCUSSION THREAD

**Discussion by swufelcj** [community/community#82758] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

GitHub team could not be created. There seems to be a problem on github.com, please try again.

**Comment by ghostinhershell** [comment]
Hi @swufelcj, we're happy you're here! You are more likely to get a useful response if you are explicit about what your issue entails, giving a few more details might help someone give you a nudge in the right direction.

**Reply by guptasuchit** [reply]
Hello, i am facing the same issue.
History:
I created an assignment 1 month back and it worked seamlessly without any issues.

Issue:
Now, following the same steps, i created another assignment - https://classroom.github.com/a/SSFri9Iu
However, when i try creating a team for this assignment, it gives the following error
<img width="1440" alt="image" src="https://github.com/community/community/assets/30351953/b48ea9ff-d19d-4837-80e2-30939d58fb2b">

can you please help me with this

More on the same,
For the previous assignment, i used a starter repository for the assignment.

When i use a starter repository for the assignment this time, it creates another clone of the repository and throws this error.
The name of the new repository is of the format (classroom_name-assignment_title-repository_name)
